src/views/HomeView.vue

```diff
@@ -57,44 +57,45 @@ onMounted(() => {
         :name="allSports.name"
       />
     </header>
-
     <aside class="sidebar">
       <SideBar />
     </aside>
     
-    <article class="body">
-      body
+    <article class="content">
+      <div class="test">
+        hola
+      </div>
     </article>
   </div>
 </template>
 
 <style lang="scss" scoped>
 .grid-container {
-    display: grid;
-    grid-template-rows: 20% auto;
-    grid-auto-columns: 240px auto;
-    grid-template-areas:
-        'sidebar header'
-        'sidebar body'
-    ;
+  display: grid;
+  height: 100%;
+  grid-template-rows: 64px auto;
+  grid-auto-columns: 240px auto;
+  grid-template-areas:
+  'sidebar header'
+  'sidebar content'
+  ;
```

Comment:
Make sure to indent when there's code continuation. Looks like you had it right before, but then un-indented it.